lesson3/_homework/task_01/script.js

```diff
@@ -0,0 +1,127 @@
+
+//                    Adapt Academy 2019 lesson 03 - JS introduction | task 01 by Donatas Dereškevičius                     
+
+
+const getZodiacSign = (month, day) => {
+
+    month = document.querySelector(".form-input__month").value;
+    day = document.querySelector(".form-input__day").value;
+
+    if  ((month > 0 && month <= 12) && (day > 0 && day <= 31)){
+
+        if (( month == 12 && day >= 22) || (month == 1 && day < 20)){
+            document.querySelector(".answer-text").innerHTML = 'Capricorn';
+    
+                document.querySelector(".wrap-title").classList.add('hide');
```

Comment:
This is a repeating code pattern which I would consider putting to a separate function.